docs/api/datetime.md

```diff
@@ -283,6 +286,22 @@ Developers can provide their own buttons for advanced custom behavior.
 
 <CustomizingButtons />
 
+## Highlighting Specific Dates
+
+Using the `highlightedDates` property, developers can style particular dates with custom text or background colors. This property can be defined as either an array of dates and their colors, or a callback that receives an ISO string and returns the colors to use.
```

Comment:
Can we call out that the default selected date style will always override custom styles? Would be good to add some reasoning around that decision too.